serverless/src/tags.ts

```diff
@@ -31,3 +31,14 @@ export function addServiceAndEnvTags(lambdas: LambdaFunction[], service: string
     }
   });
 }
+
+export function addMacroTag(lambdas: LambdaFunction[], version: string | undefined) {
+  if (!version) return;
+
+  lambdas.forEach((lambda) => {
+    const tags = lambda.properties.Tags ?? [];
+    tags.push({ Value: `v${version}`, Key: "dd_sls_macro" });
```

Comment:
Should `dd_sls_macro` be a constant?

Reply:
Good call, was following the pattern for the other tags. Going to move those to constants maybe using an enum.